app/helpers/mediums_helper.rb

```diff
@@ -1,3 +1,12 @@
 module MediumsHelper
   include PtablesHelper
+
+  def required_nfs_list
+    Operatingsystem.families.select {|family| family.constantize.require_nfs_access_to_medium }
+  end
+
+  def required_nfs?
+    required_nfs_list.include?(@medium.os_family) ||
+    @medium.operatingsystems.any?{ |obj| obj.class.require_nfs_access_to_medium }
```

Comment:
The second part of this check is redundant, you can only add operating systems that match the family